src/site/apt/usage.apt.vm

```diff
@@ -80,8 +86,24 @@ Usage
 </project>
 +---+
 
-  You can read more about which standard toolchains are available
-  {{{./toolchains/index.html}here}}.
+  You can read more about which standard toolchains are available {{{./toolchains/index.html}here}}.
+
+* The <<<toolchains.xml>>> Descriptor
+
+  Tools available on the building machine are described in <<<~/.m2/toolchains.xml>>>:
+
+  * tool <<<\<type\>>>>,
+
+  * <<<\<provides\>>>> elements to describe tool characteristics that can later be matched from plugin, when multiple
+  tools for on <<<type>>> are available,
+
+  * <<<\<configuration\>>>> elements to access the tool.
+
+  []
+
+  See {{{/ref/3.9.6/maven-core/toolchains.html}Toolchains Descriptor}} for details on its structure.
```

Comment:
This link is going to be a pain to keep up to date with new versions. You use ref/current below. Would that work?

Reply:
oh yes, I forgot to update to current: will fix, thanks